generator/src/main.rs

```diff
@@ -1926,6 +1929,7 @@ impl Parser {
 
 #[derive(Debug, Copy, Clone, Default)]
 struct StructMeta {
+    has_function_pointer: bool,
```

Comment:
Not sure if there's a better way to do this than introducing this field

Reply:
Short of generating a `Debug` impl explicitly for these cases (which I don't particularly care about), I think it's the most pragmatic solution.